pages/index.js

```diff
@@ -11,18 +11,42 @@ import {
   StyledAllFormButtonsContainer,
 } from "@/styles";
 import FilterCategory from "@/components/FilterCategory/FilterCategory";
-export default function HomePage({
-  transactions,
+import useSWR, {mutate} from "swr";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());

```

Comment:
It makes sense to extract the fetcher to the _app.js to make it globally accessible
https://github.com/neuefische/hh-web-23-5/blob/main/sessions/react-data-fetching/react-data-fetching.md#configuring-swr